tests/functional/cursorid-001.phpt

```diff
@@ -32,6 +32,7 @@ var_dump($s1 > 0);
 ===DONE===
 <?php exit(0); ?>
 --EXPECTF--
+Deprecated: MongoDB\Driver\Cursor::getId(): The method "MongoDB\Driver\Cursor::getId" will no longer return a "MongoDB\Driver\CursorId" instance in the future. Pass "true" as argument to change to the new behavior and receive a "MongoDB\BSON\Int64" instance instead. in %s
```

Comment:
If you change this test to use `Cursor::getId(true)`, you can definitely remove the string cast and extra `var_dump()`. That was likely only needed because CursorId had no comparison handler.

Dumping an Int64 and checking that it's greater than zero should be sufficient.

Reply:
Good point, updated accordingly